pkg/descheduler/evictions/evictions.go

```diff
@@ -76,6 +77,7 @@ func NewPodEvictor(
 	ignorePvcPods bool,
 	evictFailedBarePods bool,
 	metricsEnabled bool,
+	ignoreLocalPvcPods bool,
```

Comment:
Would you mind moving the `ignoreLocalPvcPods` next to `ignorePvcPods`?

Reply:
Sure, wasn't sure what the best practice was there